docs/governance/index.rst

```diff
@@ -208,3 +208,11 @@ Amendments
 Amendments or changes to this governance document can be made by a vote of the advisors. Proposed changes must be submitted to the advisors for review at least one week prior to the vote taking place. Prior to the vote, the advisors should allow for a period of discussion where the changes or amendments can be reviewed by the group. Amendments are ratified by the normal voting procedure described above.
 
 Amendments should update the version and date at the top of this document.
+
+Board of Advisors Minutes
+-------------------------
+
+.. toctree::
+    :maxdepth: 1
+
+    minutes/2019-05-15.rst
```

Comment:
I'm not sure I like how this looks appended to the ended of the governance document, but it's the only way I could figure out how to include it using the RTD sphinx theme:

<img width="1118" alt="Screenshot 2019-06-09 09 07 00" src="https://user-images.githubusercontent.com/745966/59159320-0380a200-8a96-11e9-9a64-2195b0058822.png">

Reply:
By default, the TOC is collapsed though, so maybe it's not too disruptive?